src/Symfony/Component/Serializer/Tests/Normalizer/GetSetMethodNormalizerTest.php

```diff
@@ -93,7 +91,7 @@ public function testNormalize()
             ->expects($this->once())
             ->method('normalize')
             ->with($object, 'any')
-            ->will($this->returnValue('string_object'))
+            ->willReturn('string_object')
```

Comment:
I don't know why fabbot is adding these now, but this should be done on lower branches first and reverted here

Reply:
reverted in 0dd0679362 according to #31737